layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,212 @@
+<<<<<<< Local Changes
```

Comment:
looks like some git debris got caught in here?

Reply:
noticed it just as you did :P